src/js/tabs/receiver.js

```diff
@@ -319,6 +374,31 @@ TABS.receiver.initialize = function (callback) {
                 spiRxElement.append(`<option value="${i}">${spiRxTypes[i]}</option>`);
             }
 
+            if (semver.gte(FC.CONFIG.apiVersion, API_VERSION_1_44)) {
```

Comment:
It should be msp 1.45 like firmware pr, merge this for 4.4

Reply:
I was told I shouldn't change these constants, but I will change it to 1.45 when it's added

Reply:
You need to change it to `API_VERSION_1_45` so that the configurator only executes the code inside the if-statement, if a flight controller with BF 4.4 installed is connected (which will use API 1.45).